dss-api.yml

```diff
@@ -1452,7 +1455,11 @@ paths:
         - name: directurls
           in: query
           description: >
-            When set to true, the response will contain API-specific URLs that are tied to the specified replica, for example `gs://bucket/object` or `s3://bucket/object`. This parameter is mutually exclusive with the presigned urls parameter.
+            When set to true, the response will contain API-specific URLs that are tied to the specified replica,
+             for example `gs://bucket/object` or `s3://bucket/object`. This parameter is mutually exclusive with the presigned urls parameter.
+            The use of presigned URLs is recommended for data access.
```

Comment:
Ditto comment above.

Reply:
reformatted yml, verified in presentation in Swagger UI page using http://editor.swagger.io/